src/udgl/Details.svelte

```diff
@@ -1,6 +1,9 @@
 <script>
 import { fly } from 'svelte/transition';
```

Comment:
we should remove this from `src/udgl/` and put it in `src/components/` somewhere.

Reply:
Unless I'm mistaken, it doesn't look like this or *DetailSection.svelte* is used at all. Do we plan to use them or are they artifacts of the past?

Reply:
Good catch. Let's toss em.